README.rst

```diff
@@ -35,9 +35,11 @@ To install ckanext-msal:
    config file (by default the config file is located at
    ``/etc/ckan/default/production.ini``).
 
-4. Restart CKAN. For example if you've deployed CKAN with Apache on Ubuntu::
+4. Edit `msal_config.py <https://github.com/ongov/ckanext-msal/blob/ckan_2.9.7_compatible/ckanext/msal/msal_config.py>`_ and replace the generic values with your specific credentials.
```

Comment:
this link will point to the deleted branch once merged. The documentation need work and a fix for this broken link will be part of the documentation improvement.